docs/README_sp.md

```diff
@@ -0,0 +1,195 @@
+   * Use la ubicación predeterminada o personalícela como desee, luego haga clic en **Done**
+
+1. Configurar las reglas de seguridad de Firestore:
+   * Navegue a Develop > Database > Rules en la consola de Firebase:
+     <https://console.firebase.google.com/project/$PROJECT_ID/database/firestore/rules>.
+   * Asegurese que **Cloud Firestore** esté seleccionado en el menú desplegable de la parte de arriba.
+     ![firestore rules page screenshot][]
+   * Establezca las reglas de seguridad a las que se encuentran en [`firestore/firestore.rules`][].
+
+## Desplegando la Aplicación por Primera Vez
+
+Este proyecto utiliza [Cloud Build][] y [Config Connector][] para automatizar el código y las implementaciones de infraestructura. Las instrucciones a continuación describen cómo implementar la aplicación.
+
+### 1. Configurar el proyecto de GCP
+
+Deberá iniciar los servicios y permisos requeridos por este ejemplo. La forma más fácil de hacerlo es ejecutando [bootstrap.sh][]:
+
+```bash
+./scripts/bootstrap.sh $PROJECT_ID
+```
+
+Este paso además crea un archivo llamado `env.mk` basado en [env.mk.sample](env.mk.sample).
+
+### 2. Completar las secciones TODO en `env.mk`
+
+Dirijase a los comentarios de TODOen la parte superior de `env.mk` y asegurese que los valores sean corrector.
+
+### 3. Crear un clúster de GKE
+
+Ejecutar `make cluster`
+
+### 4. Agregar un propietario verificado para el dominio
+
+Agregue la siguiente cuenta de servicio como [additional verified owner][]:
+
+`cnrm-system@${PROJECT_ID}.iam.gserviceaccount.com`
+
+donde `${PROJECT_ID}` se reemplaza por su ID de proyecto de Google Cloud.
+
+### 5. Build y desplegar
+
+Ejecute `make build-all`.
+
+## Probar la Aplicación
+
+Una vez que se despliega su aplicación, puede probarla navegando a `https://$DOMAIN`,
+donde `$DOMAIN` es el dominio personalizado que configuró `env.mk`.
+
+## Actualizar la Aplicación
+
+Ejecutar `make build-all` hará el build y desplegará la aplicación, incluidos los cambios realizados en la infraestructura. Tenga en cuenta que eliminar recursos de `infrastructure-tpl.yaml`
+no hará que se eliminen. Debe ejecutar `make delete` antes de eliminar el recurso (luego volver a implementar con `make build-all` después de eliminarlo), o eliminar manualmente el recurso con `kubectl delete`.
+
+```shell
+# construye e implementa infraestructura de back-end, frontend e KCC
+make build-all
+
+# construye e implementa solo el servicio de back-end Go
+make build-backend
+
+# construye y despliega solo la aplicación web angular frontend
+make build-webui
+```
+
+## Limpieza
+
+La ejecución de `make delete` eliminará los recursos del conector de configuración de su clúster, lo que hará que el conector de configuración elimine los recursos de GCP asociados. Sin embargo, debe eliminar manualmente su servicio Cloud Run y ​​GKE Cluster.
+
+[APIs & Services -> Credentials]: https://console.cloud.google.com/apis/credentials
```

Comment:
Many google.com URLs can be appended with `?hl=es` to present the Spanish-language UI. E.g. https://console.cloud.google.com?hl=es